### PR TITLE
New package: ardor-2.0.14

### PR DESCRIPTION
**Pull request by porky11**

### CODE REVIEW

File: srcpkgs/ardor/template [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Template file for 'ardor'
+pkgname=ardor
+version=2.0.14
+revision=1
+wrksrc="${pkgname}"
+hostmakedepends="unzip"
+depends="openjdk"
```
Comment (abenson) [inline srcpkgs/ardor/template]: Is it a hard dependency on openjdk? Could it be replaced with `virtual?java-environment` (for oraclejdk) or even `virtual?java-runtime` if it doesn't need the development stuff?

Comment (abenson) [reply srcpkgs/ardor/template]: It more than likely should be `virtual?java-runtime`

Comment (porky11) [reply srcpkgs/ardor/template]: updated this and fixed some other bug (also for nxt)

File: srcpkgs/ardor/files/ardor-tor/run [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+exec ardor-tor > /dev/null
+
```
Comment (maxice8) [inline srcpkgs/ardor/files/ardor-tor/run]: remove extra blank lines

Comment (porky11) [reply srcpkgs/ardor/files/ardor-tor/run]: btw. removed extra blank lines here too some time ago

**Comment by porky11** [comment]
@maxice8 I also did the same changes to this

File: srcpkgs/mftrace/template [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'mftrace'
+pkgname=mftrace
```
Comment (maxice8) [inline srcpkgs/mftrace/template]: Why add this package ? seems unrelated

File: srcpkgs/ardor/files/bin/ardor [hunk 4]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+cd /usr/lib/ardor && bash run.sh
```
Comment (maxice8) [inline srcpkgs/ardor/files/bin/ardor]: imo you can just the content of these binaries as the runit runfiles

File: srcpkgs/mftrace/template (hunk 3 above)
Comment (porky11) [reply srcpkgs/mftrace/template]: No Idea, what this is doing here. Maybe I made some mistakes when rebasing

File: srcpkgs/ardor/files/bin/ardor (hunk 4 above)
Comment (porky11) [reply srcpkgs/ardor/files/bin/ardor]: Yeah, the runit files are not much shorter. I just did it the same way as I did with nxt (which is basically the same (older version 1.x))
